packages/proto-build/src/lib.rs

```diff
@@ -31,7 +31,7 @@ const COSMOS_SDK_DIR: &str = "../../dependencies/cosmos-sdk/";
 const OSMOSIS_DIR: &str = "../../dependencies/osmosis/";
 
 /// A temporary directory for proto building
-const TMP_BUILD_DIR: &str = "/tmp/tmp-protobuf/";
+const TMP_BUILD_DIR: &str = "./tmp/tmp-protobuf/";
```

Comment:
wondering why do we need `.` here?

Reply:
my mistake, I had to build the types locally because I was getting permission errors on my home path, will change that back